packages/neo-one-smart-contract-compiler/src/compile/syscalls.ts

```diff
@@ -260,6 +260,8 @@ const OutputValue = new BlockchainInterface('OutputBase');
 const TransactionValue = new BlockchainInterface('TransactionBase');
 const ValidatorValue = new BlockchainInterface('ValidatorBase');
 const StorageContextValue = new BlockchainInterface('StorageContextBase');
+const EnumeratorValue = new BlockchainInterface('EnumeratorBase'); // type param overload
+const IteratorValue = new BlockchainInterface('IteratorBase');
 const StorageIteratorValue = new BlockchainInterface('StorageIteratorBase');
```

Comment:
This should be removed and replaced with `Iterator<SerializableValue, SerializableValue>` everywhere.